packages/clay-alert/stories/index.tsx

```diff
@@ -95,5 +96,20 @@ storiesOf('Components|ClayAlert', module)
 			{text('Content', 'This is an alert!')}
 		</ClayAlert>
 	))
+	.add('w/ a button', () => (
+		<ClayAlert
+			displayType="info"
+			spritemap={spritemap}
+			title="With a Button"
+		>
+			{'This is an alert with a button!'}
+
+			<ClayAlert.Footer>
+				<ClayButton.Group>
+					<ClayButton alert>{'View'}</ClayButton>
+				</ClayButton.Group>
+			</ClayAlert.Footer>
+		</ClayAlert>
+	))
```

Comment:
hey @bryceosterhaus I think we need to do more things to make the component fit the markup that @pat270 said (https://github.com/liferay/clay/issues/2765#issuecomment-569127316).